webapp/security/models.py

```diff
@@ -77,3 +77,6 @@ class Release(Base):
     version = Column(String, unique=True)
     codename = Column(String, unique=True)
     lts = Column(Boolean)
+    development = Column(Boolean)
+    release_date = Column(DateTime)
+    esm_expire_date = Column(DateTime)
```

Comment:
If we have the `release_date` can't we work out when ESM is expired?
What is `development`?

Reply:
- Well, I guess I did it this way for people who don't know that ESM expires in 5 years?
-`development ` is for unreleased Ubuntu versions that also show in CVEs: https://people.canonical.com/~ubuntu-security/cve/main.html

Reply:
What I mean is you can work out in python if the releases is `esm` if you have a release date.